fcrepo-clients/src/test/java/edu/unc/lib/dl/fcrepo4/DepositRecordIT.java

```diff
@@ -172,6 +175,21 @@ public void addPremisEventsTest() throws Exception {
 		// Check that the second event has the right type
 		assertTrue(events.get(1).getResource().hasProperty(Premis.hasEventType, Premis.Ingestion));
 	}
+	
+	public void addObjectsTest() throws Exception {
```

Comment:
The test is throwing an exception with a 400 response from fedora.  It might be because the model.createResource() calls are using weird auto-generated IDs and because those ids don't exist in fedora, but we are trying to reference them.  Or the sparql syntax could be off.  So many options!